en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,59 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+When it comes to display the value, to list the entities for instance, there is no easy way to render
+the value to the backoffice user so we have added a ``stringValue`` to ``ContentAttribute``.
+The ``stringValue`` is a projection of the ``value`` in a string.
+
+Transformers
+------------
+
+Adding a Strategy
+~~~~~~~~~~~~~~~~~
+
+Open Orchestra generate ``stringValue`` thanks to transformation strategies.
+The different already existent strategies are available in
+``OpenOrchestra\Backoffice\ValueTransformer\Strategies``.
+
+To generate a specific transformation strategy, create a service implementing the
+``OpenOrchestra\Backoffice\ValueTransformer\ValueTransformerInterface``
+and implement the required methods:
+
+- ``transform``: transform value to an HTML string
+- ``support``: return true if the transformer must be used
+- ``getName``: uses by manager as index for the transformer
+
+Then, for the service to be detected, give him the tag
+``open_orchestra_backoffice.value_transformer.strategy``:
+
+.. code-block:: yml
+
+    open_orchestra_backoffice.value_transformer.array:
+            class: %open_orchestra_backoffice.value_transformer.array.class%
+            tags:
+                - { name: open_orchestra_backoffice.value_transformer.strategy }
+
+Override a strategy
+~~~~~~~~~~~~~~~~~~~
+
+To override a strategy, add a new ``ValueTransformerInterface`` with the same ``getName`` method.
+The last one call will override the others.
```

Comment:
this is not useful and hard to grasp, you can simplify :
If you want to override a strategy, you just need to register a new ``ValueTransformerInterface`` with the same name.